### PR TITLE
Adding additional setup information

### PR DESCRIPTION
**Pull request by LiliamLeme**
Adding setup information and readme information:

Docs folder
1) JSON file 
2) Setup for testing libraries
3)link to this information in the readme

### CODE REVIEW

**Comment by LiliamLeme** [comment]
Adding setup information and readme information:

Docs folder
1) JSON file 
2) Setup for testing libraries
3)link to this information in the readme

File: Docs/EnviromnentSetup/JSON_pipeline_files.md
Comment (arjendev) [reply Docs/EnviromnentSetup/JSON_pipeline_files.md:3]: Fabric: https://learn.microsoft.com/en-us/fabric/cicd/git-integration/git-integration-process

ADF: https://learn.microsoft.com/en-us/azure/data-factory/source-control

File: Docs/EnviromnentSetup/Unittest_setup.md
Comment (arjendev) [inline Docs/EnviromnentSetup/Unittest_setup.md:14]: Can you host these assets under docs/assets in this repository so we contain everything in the same repo?

File: Docs/EnviromnentSetup/Unittest_setup.md
Comment (arjendev) [inline Docs/EnviromnentSetup/Unittest_setup.md:40]: Poetry is a package management library and not a testing library. Perhaps you can mention pytest or unittest here and then mention poetry later when you are talking about package management.

File: README.md
Comment (arjendev) [inline README.md:39]: do we need a comma here now that you removed the "and"?

File: README.md
Comment (arjendev) [inline README.md:44]: Could we keep the non-opiniated readme here and then link to a getting-started.md which contains the opiniated getting started? :)

File: README.md
Comment (arjendev) [inline README.md:92]: Once getting started with examples moved (see prev comment), I think this # can be removed again

**Review by arjendev — COMMENTED** [review]
Thanks @LiliamLeme !!

Sharing some initial comments, will do another review once they have been addressed :)

File: Docs/environment_setup/JSON_pipeline_files.md
Comment (ydaponte) [inline Docs/environment_setup/JSON_pipeline_files.md:35]: @LiliamLeme - let's avoid to point to personal repos. I think it is better to create an "Images" folder in the current repo because we all will need to add images of the example pipelines

**Review by ydaponte — APPROVED** [review]
@LiliamLeme - I'm approving not to block the PR - but please address the image comment before merging. Other than that LGTM. Thanks!

**Review by arjendev — CHANGES_REQUESTED** [review]
I'd like to review the docs before going to main, so will mark the PR as "request changes" for now.

File: README.md
Comment (arjendev) [inline README.md:45]: Can you use relative paths for all the images? Then it will always resolve the images from the current repository / branch.

File: README.md
Comment (arjendev) [inline README.md:50]: /nit: single :

File: README.md
Comment (arjendev) [inline README.md:92]: Why did you change this section to bold?

File: README.md
Comment (arjendev) [reply README.md]: Re-opening this conversation as your code example is still in the main readme :)

File: docs/environment_setup/images/DownloadSupportFiles.png
Comment (ydaponte) [inline docs/environment_setup/images/DownloadSupportFiles.png:1]: @LiliamLeme - can we follow the same name convention for the files. Look at the list under images, they are almost all lower case and using "_" - with that logic this file should be named: download_support_files.png ( and all the references to the file should be changed as well if any)

File: docs/environment_setup/images/New_vs_project.png
Comment (ydaponte) [inline docs/environment_setup/images/New_vs_project.png:1]: File name should be new_vs_project.png ( and all the references to the file should be changed as well if any)

File: docs/environment_setup/Unittest_setup.md
Comment (ydaponte) [inline docs/environment_setup/Unittest_setup.md:1]: File name should be unit_test_setup.md ( and all the references to the file should be changed as well if any)

File: docs/environment_setup/JSON_pipeline_files.md
Comment (ydaponte) [inline docs/environment_setup/JSON_pipeline_files.md:1]: File name should be changed to json_pipeline_files.md ( and all the references to the file should be changed as well if any)

File: README.md
Comment (ydaponte) [inline README.md:55]: Change file name aligned with the other comment to json_pipeline_files,md

File: README.md
Comment (ydaponte) [inline README.md:46]: Change reference to file to unit_test_setup.md aligned with previous comment on the file

File: docs/environment_setup/JSON_pipeline_files.md
Comment (ydaponte) [inline docs/environment_setup/JSON_pipeline_files.md:8]: As a best practice, we normally remove the localization from the link so we are inclusive of all locations.

File: docs/environment_setup/JSON_pipeline_files.md
Comment (ydaponte) [inline docs/environment_setup/JSON_pipeline_files.md:18]:
```suggestion
Refer to the docs if needed: [Azure Data Factory Documentation - Azure Data Factory | Microsoft Learn](https://learn.microsoft.com/azure/data-factory/)
```

File: docs/environment_setup/JSON_pipeline_files.md
Comment (ydaponte) [inline docs/environment_setup/JSON_pipeline_files.md:8]:
```suggestion
[Git integration process - Microsoft Fabric | Microsoft Learn](https://learn.microsoft.com/fabric/cicd/git-integration/git-integration-process)
```

File: docs/environment_setup/Unittest_setup.md
Comment (ydaponte) [inline docs/environment_setup/Unittest_setup.md:16]:
```suggestion
![image](/docs/environment_setup/images/new_vs_project.png)
```

File: README.md
Comment (ydaponte) [inline README.md:46]:
```suggestion
   [docs_Setup](/docs/environment_setup/unit_test_setup.md)
```

File: README.md
Comment (ydaponte) [inline README.md:55]:
```suggestion
   [docs Json](/docs/environment_setup/json_pipeline_files.md)
```

File: docs/environment_setup/JSON_pipeline_files.md
Comment (ydaponte) [inline docs/environment_setup/JSON_pipeline_files.md:11]:
```suggestion
[Source control - Azure Data Factory | Microsoft Learn](https://learn.microsoft.com/azure/data-factory/source-control)
```

File: README.md
Comment (ydaponte) [inline README.md:91]:
```suggestion
## Features - Examples
```

**Review by ydaponte — CHANGES_REQUESTED** [review]
Left some comments - details but for consistency reasons we should address them. Also made suggestions that you can just accept to be easier to fix. Thanks!

File: README.md
Comment (arjendev) [inline README.md:48]: Fix numbering, it skips 2, 4 and 6 now.

File: README.md
Comment (arjendev) [inline README.md:59]: You closed the comment twice that asks to move this section to getting-started.md and then you can introduce a link to there.

File: docs/environment_setup/Unittest_setup.md
Comment (arjendev) [inline docs/environment_setup/Unittest_setup.md]: Numbering inconsistent. 3 is skipped.

File: docs/environment_setup/Unittest_setup.md
Comment (arjendev) [inline docs/environment_setup/Unittest_setup.md]: Inconsistent numbering. 2 is skipped.

File: README.md
Comment (ydaponte) [reply README.md:48]: @LiliamLeme - my apologies, the suggestion introduced a mistake in the numbering. Please review all the lists.

File: docs/environment_setup/unit_test_setup.md
Comment (ydaponte) [inline docs/environment_setup/unit_test_setup.md:19]:
```suggestion
2. Pip install the following libraries from the terminal - data-factory-testing-framework:
```

**Review by arjendev — APPROVED** [review]
Looks great Liliam, thank you for your contribution and for being so patient with me!

**Review by ydaponte — APPROVED** [review]
LGTM! Thank you!

**Comment by LiliamLeme** [comment]
mering readme

**Pull request merged**
